src/app.js

```
@@ -1 +1,66 @@
+/* eslint-disable no-console */
 // write code here
+const fs = require('fs');
+const path = require('path');
+
+function main() {
+  const [source, dest] = process.argv.slice(2);
+
+  if (source === undefined || dest === undefined) {
+    console.error('Specify params');
+
+    return;
+  }
+
+  if (!fs.existsSync(source)) {
+    console.error('Source file does not exist');
+
+    return;
+  }
+
+  if (source === dest) {
+    return;
+  }
```

This condition checks if the source and destination are the same, but it silently returns without any message. It might be helpful to log a message indicating that the source and destination are the same, and no action is taken.